offchain/danaswap-ui/flake-module.nix

```diff
@@ -6,25 +6,43 @@
       inherit (config) cat-lib dream2nix;
       ui =
         (dream2nix.lib.makeOutputs { source = ./.; settings = [{ subsystemInfo.nodejs = 16; }]; }).packages.danaswap-ui;
+
+      fa_sprite_util = pkgs.callPackage ./tools/fa_sprite_util/default.nix { };
+
+      iconsToKeep = [
+        # brands
+        "discord"
+        "linkedin"
+        "medium"
+        "reddit"
+        "telegram"
+        "twitter"
+
+        # solid
+        "angle-down"
+        "angle-up"
+        "right-left"
+        "sun"
+      ];
+
       font-awesome-sprites =
-        # TODO: figure out which icons we _need_, and strip the rest as almost
-        # all of these icons are unused dead weight
-        # NOTE: "--enable-comment-stripping" can’t be added due to licensing
-        # (CC BY 4.0 requires attribution). To enable, the license data
-        # should be moved to the proper metadata elements and namespaces (I’m
-        # unsure why Font Awesome didn’t do this to begin with)
+        let
+          fontAwesomeVersion = "6.2.0";
+          fontAwesomeYear = "2022";
+          iconsFlags = builtins.concatStringsSep " " (builtins.map (i: "--icon=\"${i}\"") iconsToKeep);
```

Comment:
replace the concatStringsSep with [escapeShellArgs](https://teu5us.github.io/nix-lib.html#lib.strings.escapeshellargs)